influxdb_plugin.py

```diff
@@ -181,6 +183,35 @@ def _process_stat_list(stat_value, fields, tags, prefix=""):
                 _add_field(fields, item_name, list_value, value_type)
 
 
+def _influxdb_points_from_stat(stat_time, tags, stat_key, stat_value):
+    """
+    Create InfluxDB points/measurements from the stat query result.
+    """
+    points = []
+    point_tags = tags.copy()
+    fields = []
+    stat_value_type = type(stat_value)
+    if stat_value_type == list:
+        for stat in stat_value:
+            (fields, point_tags) = _influxdb_point_from_stat(
+                stat_time, tags, stat_key, stat)
+            points.append(
+                _build_influxdb_point(
+                    stat_time, point_tags, stat_key, fields))
```

Comment:
If the point_tags returned by each call to _influxdb_point_from_stat are not entirely unique then only the last point in points will get stored. However, this scenario seems unlikely. I'm thinking perhaps you could do a test case where you verify that for the current set stats in the example config file this never happens. Solutions #2 and #3 appear to be ways to identify if this situation happens. Although, I'm thinking that we just do this as test to verify that we aren't causing any problems with our current set of stats. I'd rather not include this verification code in the final product since it seems pretty innefficient thing to do and most likely not necessary - just want to verify that the assumption is true.